schemas/context/media-timed-asset-view-details.schema.json

```diff
@@ -74,10 +89,15 @@
           "title": "Quality of experience",
           "$ref": "https://ns.adobe.com/xdm/context/media-timed-qoe",
           "description": "Timed media properties related to the content playback's quality of experience (QoE)."
+        },
+        "xdm:downloadedPlayback": {
```

Comment:
Isn't there a pre-existing media grammar that could be used?  Do we really need to invent a new one??

Reply:
This is something that we defined and computed in Video Analytics to describe what type of content was played. As such, it doesn't mean anything generic that could be found in a pre-existing grammar.